docs/glossary.txt

```diff
@@ -105,7 +105,7 @@ sample: a basic conceptual unit of a surface, a sample corresponds to a single
 set of component values.
 
 storage type: an enumerated value selecting the internal layout of a surface
-[as seen on the high level], or pages / tiled regions [low level].
```

Comment:
tiled was right here - it refers to pre-G80 tiling hardware, which worked very much unlike modern tiling aka blocklinear surfaces.

In short: on G80+, the pitch/blocklinear format is visible to whatever is accessing the memory (thus you have to set it in all the methods, etc.), but pre-G80 there are "tiled regions" which look like pitch format to memory clients, but are stored in an optimised way by the memory controller. And nv definitely calls these "tiled".